### ISSUE TITLE
[Bug]: Property binding doesn't seem to work (Angular)

### ISSUE THREAD

**Issue by MaximeRobion** [add2cal/add-to-calendar-button#517]
### What happened?

When there is no data binding, it works fine.

HTML Template:
```
    <add-to-calendar-button
        [attr.name]="addCalendarTitle"
        startDate="2023-12-20"
        startTime="10:15"
        endTime="17:45"
        timeZone="Europe/London"
        location="World Wide Web"
        options="'Apple','Google','iCal','Outlook.com'"
        listStyle="overlay"
        hideIconButton
        size="3"
    ></add-to-calendar-button>
```

JS:
```
import {Component, EventEmitter, Input, Output, SimpleChanges, ViewEncapsulation} from "@angular/core";
import {Viewing} from "../../../models/tenant-find/viewing";
import 'add-to-calendar-button';

@Component ({
    selector: 'viewing-view-manager',
    templateUrl: './viewing-view-manager.component.html',
    encapsulation: ViewEncapsulation.None
})
export class ViewingViewManagerComponent {
    @Input() viewing: Viewing;
    addCalendarTitle: string = 'Default Title';

    ngOnInit(){
        this.setUpAddCalendarVar()
    }

    setUpAddCalendarVar() {
        this.addCalendarTitle ='Viewing at ' + this.viewing.propertySummary.name
    }
}

```

Error message: Error: Uncaught (in promise): TypeError: Attempted to assign to readonly property.

### System & Environment

Angular 15

**Comment by jekuer** [comment]
Fixed with v2.5.1

**Comment by MaximeRobion** [comment]
Working now! cheers :) Thanks for the fast response!
